index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AwesomeBooks with ES6</title>
+    <link rel="stylesheet" href="index.css">
+    
+</head>
+<body>
+    <header>
+        <h2 class="mylogo">Awesomebookstore</h2>
+        <nav class="links">
+            <ul>
+                <li>List</li>
+                <li>Add</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class="date" id="currdate"></div> 
+        <section class="book">        
+        <h2>All Awesome Books List</h2>  
+            <div class="allbooks">        
+            </div>
+         
+        </section>
+        <section class="addbook hide">
+           
+                <h3>Add New books</h3>
+           
+                <form action="" method="post" id="form">
+                <input type="text" name="" id="title" placeholder="Add Title" required>
+                <input type="text" name="" id="author" placeholder="Add Author" required>
+                <button type="button" id="add">Add</button>
+            </form>
+        </section>
+        <section class="contat hide">
+            <h2>Contact Info</h2>    
+                <h4>Do you have any questions or just want to say "Hello"?<br>You can reach us at:-</h4>
+                <ul>
+                    <li>Name: Supreeti Kushwaha</li>
+                    <li>Contact: +919106438868</li>
+                    <li>Email: sksupreetk11@gmail.com</li>
+                    <li>Address:A-234 ,Greater Heights, Gujarat,India</li>
+                </ul>          
+        </section>
+    </main>
+    <footer>
+            <h4>Copyright</h4>
+             <h3>By Supreeti</h3>
+    </footer>
```

Comment:
### OPTIONAL

- [ ] It would be better for your footer to be fixed to the bottom of your page, Currently the footer is not fixed to the bottom 👍🏿